torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1823,12 +1821,12 @@ int dictUpdate(Stack& stack) {
 
 int dictItems(Stack& stack) {
   auto dict = pop(stack).toGenericDict();
-  std::vector<IValue> items;
+  auto items = c10::impl::GenericList(c10::impl::deprecatedUntypedList());
```

Comment:
we can inherit the type from dict, right?

Reply:
yes, going to do it in a separate PR.